pipelines/stack_os_matrix.groovy

```diff
@@ -27,9 +27,10 @@ notify.wrap {
   ])
 
   def buildParams = [
-    LSST_REFS:      REFS,
-    LSST_PRODUCTS:  PRODUCTS,
-    LSST_BUILD_DOCS: BUILD_DOCS,
+    LSST_REFS:              REFS,
+    LSST_PRODUCTS:          PRODUCTS,
+    LSST_BUILD_DOCS:        BUILD_DOCS,
+    LSST_NO_BINARY_FETCH:   false,
```

Comment:
Are you sure this isn't overriding the other `true` values you've now inserted?  It looks to me like it might.